src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -1039,25 +1039,55 @@ There are various error codes for different error scenarios. The following table
-  * Tracked Entity
-      * Enrollment
-        * Event
-            * DataValue -> Link
-            * TrackedEntityAttributeValue -> Link
-      * TrackedEntityAttributesValue -> Link
-      * TrackedEntityAttributeValue
-        * Value Types
+While importing data using the tracker importer, a series of validations are performed to ensure the validity of the data. This section will describe some of the different types of validation performed, to provide a better understanding if validation fails for your import.
 
+#### Required properties
+Each of the tracker objects has a few required properties that needs to be present when importing data. For an exhaustive list of required properties, have a look at the [Tracker Object section](#webapi_nti_tracker_objects).
+
+When validating required properties, we are usually talking about references to other data or metadata. In these cases there are 3 main criteria:
+1. The reference is present, and not null, in the payload.
+2. The reference points to the right type of data and exists in the database
+3. The user have access to see the reference
+
+If the first condition fails, the import will fail with a message about a missing reference. However, if the reference points to something that doesnt exists, or which the user dont have access to, both cases will result in a message about the reference not being found.
+
+#### Formats
+Some of the properties of tracker objects require a specific format. When importing data, each of these properties are validated against the expected format, and will return different errors depending on which property has a bad format. Some examples of properties that are validated this way:
+- UIDs (These cover all references to other data or metadata in DHIS2.)
+- Dates
+- Geometry (The coordinates must match the format as specified by it's type)
+
+#### User access
+All data imported will be validated based on the metadata of the data imported (Sharing) and the organisation unit of the data imported (Organisation Unit Scopes). More information about sharing and organisation unit scopes can be found in the following sections.
+
+Sharing is validated at the same time as references are looked up. That means a references the user cannot access, will be treated as if it doesn't exist. Sharing is checked for each metadata referenced in the data being imported.
+
+Organisation units on the other hand, serve a dual purpose. Primarily, it will make sure that data can only be imported when it's imported for an organisation unit the user have within their "capture scope". Secondly, organisation units are also used to restrict what programs are available. That means if you are trying to import data for an organisation unit which does not have access to the program you are importing, the import will be invalid.
+
+User's with the `ALL` authority transcends, in these cases, validations related to the user, but not programs.
```

Comment:
I don't fully understand what you mean here. Maybe add a sentence or two explaining it? User's - users?

Reply:
Completely agree. I was a bit unsure how to easily explain it, but I see now it's a bit too compact. Will make it a bit more clear :)